### PR TITLE
Remove time feature wrapper by replacing it with a dummy wrapper

### PR DESCRIPTION
**Pull request by dtch1997**
By default, SB3 zoo PPO on A1GymEnv includes an environment wrapper called TimeFeatureWrapper. 
This is printed in stdout at the start of training: 

```
========== A1GymEnv-v0 ==========
Seed: 2604389549
Default hyperparameters for environment (ones being tuned will be overridden):
OrderedDict([('batch_size', 128),
             ('clip_range', 0.4),
             ('ent_coef', 0.0),
             ('env_wrapper', 'sb3_contrib.common.wrappers.TimeFeatureWrapper'),
             ('gae_lambda', 0.9),
             ('gamma', 0.99),
             ('learning_rate', 3e-05),
             ('max_grad_norm', 0.5),
             ('n_envs', 16),
             ('n_epochs', 20),
             ('n_steps', 512),
             ('n_timesteps', 2000000.0),
             ('normalize', True),
             ('policy', 'MlpPolicy'),
             ('policy_kwargs',
              'dict(log_std_init=-2, ortho_init=False, activation_fn=nn.ReLU, '
              'net_arch=[dict(pi=[256, 256], vf=[256, 256])] )'),
             ('sde_sample_freq', 4),
             ('use_sde', True),
             ('vf_coef', 0.5)])
```

TimeFeatureWrapper simply adds a 1-dim feature of the number of steps remaining in the environment (see https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/blob/master/sb3_contrib/common/wrappers/time_feature.py). 

However it's not appropriate to include this because our agent should not depend on the time left in the environment (which is entirely artificial anyway). 

The PR fixes this by creating a dummy wrapper that does nothing and making that the default wrapper instead of TimeFeatureWrapper.

### CODE REVIEW

**Comment by dtch1997** [comment]
@jennyzzt do you think this makes sense?

**Comment by dtch1997** [comment]
After implementing this, the default feature wrapper has been changed as can be seen in STDOUt during training: 
```
========== A1GymEnv-v0 ==========
Seed: 74011589
Default hyperparameters for environment (ones being tuned will be overridden):
OrderedDict([('batch_size', 128),
             ('clip_range', 0.4),
             ('ent_coef', 0.0),
             ('env_wrapper', 'utils.wrappers.DummyWrapper'),
             ('gae_lambda', 0.9),
             ('gamma', 0.99),
             ('learning_rate', 3e-05),
             ('max_grad_norm', 0.5),
             ('n_envs', 16),
             ('n_epochs', 20),
             ('n_steps', 512),
             ('n_timesteps', 2000000.0),
             ('normalize', True),
             ('policy', 'MlpPolicy'),
             ('policy_kwargs',
              'dict(log_std_init=-2, ortho_init=False, activation_fn=nn.ReLU, '
              'net_arch=[dict(pi=[256, 256], vf=[256, 256])], '
              'features_extractor_class=A1GymEnvCombinedExtractor )'),
             ('sde_sample_freq', 4),
             ('use_sde', True),
             ('vf_coef', 0.5)])
```
